docs/dev/development.adoc

```diff
@@ -333,7 +340,7 @@ There are some test environment variable that helps to get more control over the
 
 *Running integration tests:*
 
-By default, tests are run against the `odo` binary placed in the PATH which is created by command `make`. Integration tests can be run in two (parallel and sequential) ways. To control the parallel run use environment variable `TEST_EXEC_NODES`. For example component test can be run
+Login to the cluster using cluster credentials for running test on 3.11 cluster. For example `odo login -u <user_name> -p <password> <server_URL>`. However `make configure-installer-tests-cluster` does login for running the test on 4.* cluster. By default, tests are run against the `odo` binary placed in the PATH which is created by command `make`. Integration tests can be run in two (parallel and sequential) ways. To control the parallel run use environment variable `TEST_EXEC_NODES`. For example component test can be run
```

Comment:
replacing `test` with `tests` makes more sense. Except `For example component test can be run`

Reply:
Done